modules/global-search/src/main/java/com/ibm/cloud/platform_services/global_search/v2/GlobalSearch.java

```diff
@@ -12,7 +12,7 @@
  */
 
 /*
- * IBM OpenAPI SDK Code Generator Version: 3.79.0-2eb6af3d-20230905-174838
+ * IBM OpenAPI SDK Code Generator Version: 3.82.1-2082d402-20231115-195014
```

Comment:
Does is seem reasonable that your generated SDK code did not change other than the version of the generator???
If this is correct, then there's probably no reason to deliver this change.

Reply:
Make sure you are following the instructions in https://github.com/IBM/platform-services-java-sdk/blob/main/update_service.md